CMakeLists.txt

```diff
@@ -58,19 +58,21 @@ set_default_install_dirs()
 # Options and Variants ########################################################
 #
 include(CMakeDependentOption)
-option(WarpX_APP           "Build the WarpX executable application"     ON)
-option(WarpX_ASCENT        "Ascent in situ diagnostics"                 OFF)
-option(WarpX_EB            "Embedded boundary support"                  OFF)
+option(WarpX_APP           "Build the WarpX executable application"                                         ON)
+option(WarpX_ASCENT        "Ascent in situ diagnostics"                                                    OFF)
+option(WarpX_EB            "Embedded boundary support"                                                     OFF)
 cmake_dependent_option(WarpX_GPUCLOCK
-                           "Add GPU kernel timers (cost function)"      ON
-                           "WarpX_COMPUTE STREQUAL CUDA OR WarpX_COMPUTE STREQUAL HIP" OFF)
-option(WarpX_LIB           "Build WarpX as a shared library"            OFF)
-option(WarpX_MPI           "Multi-node support (message-passing)"       ON)
-option(WarpX_OPENPMD       "openPMD I/O (HDF5, ADIOS)"                  ON)
-option(WarpX_PSATD         "spectral solver support"                    OFF)
-option(WarpX_SENSEI        "SENSEI in situ diagnostics"                 OFF)
-option(WarpX_QED           "QED support (requires PICSAR)"                    ON)
-option(WarpX_QED_TABLE_GEN "QED table generation (requires PICSAR and Boost)" OFF)
+                           "Add GPU kernel timers (cost function)"                                          ON
+                           "WarpX_COMPUTE STREQUAL CUDA OR WarpX_COMPUTE STREQUAL HIP"                     OFF)
+option(WarpX_LIB           "Build WarpX as a shared library"                                               OFF)
+option(WarpX_MPI           "Multi-node support (message-passing)"                                           ON)
+option(WarpX_OPENPMD       "openPMD I/O (HDF5, ADIOS)"                                                      ON)
+option(WarpX_PSATD         "spectral solver support"                                                       OFF)
+option(WarpX_SENSEI        "SENSEI in situ diagnostics"                                                    OFF)
+option(WarpX_QED           "QED support (requires PICSAR)"                                                  ON)
+option(WarpX_QED_TABLE_GEN "QED table generation support enabled in WarpX(requires PICSAR and Boost)"      OFF)
+option(WarpX_QED_TOOLS     "Build external tool to generate QED lookup tables (requires PICSAR and Boost)" OFF)
```

Comment:
Please add the new option to `Docs/` in the table in `cmake.rst`

Reply:
done!